staff-registration.php

```diff
@@ -0,0 +1,104 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Last Name</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="staff-registration-input" id="lName">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Email</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="email" class="staff-registration-input" id="email">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Password</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="password" class="staff-registration-input" id="password">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Role</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <select name="role" id="role" class="staff-registration-input">
+                                    <option value="" selected>Select here</option>
+                                    <option value="owner">Owner</option>
+                                    <option value="driver">Driver</option>
+                                    <option value="guide">Guide</option>
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Contact</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="staff-registration-input" id="contact">
```

Comment:
Change this input field type to "mobile"

Reply:
OK